### ISSUE TITLE
`set priority` appears broken

### ISSUE THREAD

**Issue by selsta** [monero-project/monero#9221]
```
[wallet 431t1S]: set priority 2
Wallet password: 
[wallet 431t1S]: set
priority = 2 (normal)
[wallet 431t1S]: address
0  431t1SuHXJNE3U5GhqoGCw2t5XpAZRwyX2h3AfG5BKzkZuYWrVaEyyRdHGaXUMLhiCAc5mF5EaLN9cqie64a5PA3TXTiw3g  Primary address (used)
[wallet 431t1S]: transfer 431t1S 0.01
Wallet password: 
There is currently a 37 block backlog at that fee level. Is this okay?  (Y/Yes/N/No): n
Error: transaction cancelled.
```

**Comment by SChernykh** [comment]
It happens because of this line: https://github.com/monero-project/monero/blob/master/src/simplewallet/simplewallet.cpp#L6320
```
priority = m_wallet->adjust_priority(priority);
```
So #9219 should fix this

**Comment by SChernykh** [comment]
After looking at the code a bit more, shouldn't this line assign to the default priority, not 0? https://github.com/monero-project/monero/blob/master/src/simplewallet/simplewallet.cpp#L6316

```
uint32 priority = m_wallet->get_default_priority();
```

**Comment by SChernykh** [comment]
@selsta you can just add the one-line fix above to #9219
